engine/src/main/java/com/sandboni/core/engine/finder/bcel/visitors/http/SpringControllerMethodVisitor.java

```diff
@@ -32,14 +33,14 @@ public SpringControllerMethodVisitor(Method m, JavaClass jc, Context c, boolean
     }
 
     public void start() {
-        AnnotationEntry requestMapping = getAnnotation(javaClass.getConstantPool(), method::getAnnotationEntries, getAvailableRequestMappingAnnotations());
+        Optional<AnnotationEntry> requestMapping = getAnnotation(javaClass.getConstantPool(), method::getAnnotationEntries, getAvailableRequestMappingAnnotations());
 
-        if (requestMapping != null) {
-            String valueAnnotation = getAnnotationParameter(requestMapping, "value", "path");
+        if (requestMapping.isPresent()) {
```

Comment:
same comment here -- about isPresent vs ifPresent/map -- especially, given that you call optionalValue.get() multiple times. In this case better to abstract and call a method   doSmth(value)

Reply:
Noted for future improvement!